project.clj

```diff
@@ -0,0 +1,15 @@
+(defproject borda-clj "0.0.1"
+  :description "Clojure library for interacting with Borda"
+  :url "https://github.com/getlantern/borda-clj"
+  :license {:name "Apache Version 2.0"
+            :url "https://www.apache.org/licenses/LICENSE-2.0"}
+  :dependencies [[org.clojure/clojure "1.9.0"]
+                 [aleph "0.4.4"]
+                 [io.netty/netty-all "4.1.11.Final"]
```

Comment:
It probably doesn't matter when you only use the client library from Aleph, but whenever I use Aleph/Netty I reflexively add the following dependency, which gets you a better TLS implementation:

https://github.com/getlantern/ops-panel/blob/master/build.boot#L44

Reply:
Done!